frontend-react/src/pages/submissions/SubmissionDetails.tsx

```diff
@@ -104,6 +104,20 @@ function SubmissionDetailsContent() {
     );
     const submissionDate = generateSubmissionDate(actionDetails.submittedAt);
 
+    /* Conditional title strings */
+    const preTitle = `${
+        actionDetails.submitter
+    } ${actionDetails.topic.toUpperCase()} Submissions`;
+    const titleString: string = submissionDate
+        ? `${submissionDate.dateString} ${submissionDate.timeString}`
+        : "Date and Time parsing error";
+
+    /* Only used when externalName is present */
+    const titleWithFilename: string | undefined =
+        actionDetails.externalName !== ""
+            ? `${titleString} - ${actionDetails.externalName}`
+            : undefined;
```

Comment:
The `undefined` path seems like over complexity?
Maybe just have it default to `titleString` instead of undefined.

Then below this:
```tsx
title={
  titleWithFilename ? titleWithFilename : titleString
  }
```
becomes:
`title={titleWithFilename}`

Reply:
I already needed separate checks to ensure both `submissionDate` was properly generated and returned for `titleString`, and `actionDetails.externalName` wasn't the default empty string for `titleWithFilename`. I felt those variables together did look a tad bulky **but** having that check in the `Title` tag explains the reason why each exists, so I felt okay keeping it. Always showing `titleWithFilename` would technically be a lie (when read in english) because we're just aliasing `titleString` to that variable.

Reply:
Sounds like you thought it through! Resolved.